modules/programs/rofi.nix

```diff
@@ -167,7 +166,7 @@ let
     description = "Rasi literal string";
   };
 
-  themeType = with types; attrsOf configType;
+  themeType = with types; (attrsOf (either configType primitive));
```

Comment:
Hope there isn't anything more crazy with this format, like an `attrsOf attrsOf attrsOf ...`, or we will need to refactor those types soon. I don't think so since this is based on CSS, but you never know.

Reply:
IMO using attribute sets to represent CSS is a bad idea.

I think it should only allow `types.lines` for simplicity and if someone wants to order the lines definition they can use `mkOrder`.

Reply:
This is not a CSS though, it as CSS based format and I don't think it have anything crazier than this.

If someone has an example that this module doesn't support right now I could convert to lines, but it would be against my original reason of creating this PR (that is, supporting theme using a structured format).

Reply:
But looking at the examples [in this repository](https://github.com/davatorium/rofi-themes), I don't think there is a necessity to change the whole module to use lines.

Also, there is already an option to using a path for a theme, so it should cover the cases where the user wants to use text instead of a structured format.